rules/windows/file/file_event/file_event_win_powershell_exploit_scripts.yml

```diff
@@ -145,12 +147,14 @@ detection:
             - '\Invoke-Get-RBCD-Threaded.ps1'
             - '\Invoke-Gopher.ps1'
             - '\Invoke-Grouper2.ps1'
+            - '\Invoke-Grouper3.ps1'
             - '\Invoke-HandleKatz.ps1'
             - '\Invoke-Internalmonologue.ps1'
-            - '\Invoke-KrbRelayUp.ps1'
```

Comment:
Why did you remove this?

Reply:
Hey,

https://github.com/S3cur3Th1sSh1t/PowerSharpPack/commit/3d7a77520deac7c1731a8a97b99da3b0b4792a58

Reply:
Yeah but the script is still valid and can be downloaded from older commits and other websites. So let's keep it ;)

Reply:
sure no problem fixed